docker/build_scripts/install-libxcrypt.sh

```diff
@@ -14,6 +14,10 @@ if [ "$BASE_POLICY" == "musllinux" ]; then
 	echo "Skip libxcrypt installation on musllinux"
 	exit 0
 fi
+if [ "${AUDITWHEEL_POLICY}" == "manylinux_2_28" ]; then
```

Comment:
```suggestion
elif [ "${AUDITWHEEL_POLICY}" == "manylinux_2_28" ]; then
```
I presume both musllinux and manylinux can't be in use simultaneously.